.idea/Side-Hustle.iml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean to add all these .idea files?
for the very least, they should be their own commit, seems unrelated to this commit
but IMHO, should not be part of project at all

Reply:
I removed them @yuvalk .
They were added by accident in one of the initial PR's.

Reply:
I still see them

Reply:
![image](https://user-images.githubusercontent.com/44326706/142995450-07f64d1c-3028-4d93-8f02-48ac7ddf0985.png)

@shayakrach 
Any file in .idea was deleted.

Reply:
navigate to
https://github.com/redhat-beyond/Side-Hustle/pull/19/files
and you'll see there are 14 files.
 many are from the .idea folder
 you need to remove them from the commit.
 not locally
 it is best to add .idea to the project .gitinore file (I think you've done that)

Reply:
@yuvalk 
But the snapshot was taken from https://github.com/redhat-beyond/Side-Hustle/pull/19/files. 

I removed them locally and update that they removed. 
Is there any other solution?